### PR TITLE
HomepageSanity test cases added

### PR DESCRIPTION
**Pull request by PavanKumarN8**
@sigadamvenkata 

Homepage test cases added.

### CODE REVIEW

**Comment by adobepavan** [comment]
@sigadamvenkata

Homepage test cases added.

**Comment by sigadamvenkata** [comment]
this PR is for the Gnav availability checks on all 90 locales @skumar09 @JackySun9 @Dli3

File: package-lock.json [hunk 1]
```diff
@@ -1677,9 +1677,9 @@
       }
     },
```
[the remark this answers is not in the document]
Comment (adobepavan) [reply package-lock.json:1678]: Okay, sure.

File: configs/feds.config.js [hunk 2]
```diff
@@ -1,72 +0,0 @@
-// @ts-check
-const { devices } = require('@playwright/test');
```
[the remark this answers is not in the document]
Comment (adobepavan) [reply configs/feds.config.js:2]: I have this file on my system. I will push it in the next PR.

File: tests/feds/homePageSanity.test.js [hunk 3]
```diff
@@ -0,0 +1,8271 @@
+import { expect, test } from "@playwright/test";
+import { features } from "../../features/feds/homePageSanity.spec";
+import HomePageSanity from "../../selectors/feds/feds.homepagesanity.page.js"
+
+test.describe('Home Page Component test suite', () => {
```
[the remark this answers is not in the document]
Comment (adobepavan) [reply tests/feds/homePageSanity.test.js:5]: Okay, sure.

File: selectors/feds/feds.homepagesanity.page.js [hunk 4]
```diff
@@ -0,0 +1,4349 @@
...
+        //Gnav Elements
+        this.gnav = page.locator('#feds-nav-wrapper>div.feds-nav');
+        this.gnavCC = this.gnav.locator('section.feds-navItem > button').nth(0);
+        this.gnavDC = this.gnav.locator('section.feds-navItem > button').nth(1);
+        this.gnavEC = this.gnav.locator('section.feds-navItem > button').nth(2);
+        this.gnavHelpX = this.gnav.locator('section.feds-navItem > button').nth(3);
+
+        // 1) Creative Cloud Elements
+        //Argentina
+        this.whatIsCC = page.locator("//a[contains(@daa-ll,'Qué es Creative Cloud')]");
```
[the remark this answers is not in the document]
Comment (adobepavan) [reply selectors/feds/feds.homepagesanity.page.js:26]: We are checking the presence of the link text.

**Pull request closed without merging**